INSTALL.rst

```diff
@@ -176,13 +147,9 @@ Matplotlib requires a large number of dependencies:
   * `pytz <http://pytz.sourceforge.net/>`__
   * FreeType (>= 2.3)
   * `cycler <http://matplotlib.org/cycler/>`__ (>= 0.10.0)
-  * `six <https://pypi.python.org/pypi/six>`_
```

Comment:
Shouldn't this stay until six is completly removed from the code base

Reply:
fair, I was being optimistic.